src/pages/HomePage/HomePage.jsx

```diff
@@ -25,17 +29,86 @@ const HomePage = () => {
       {/* Featured category section */}
       <h1 className='featured-header'>Featured Categories</h1>
       <div className='featured-categories'>
-        <FeaturedCategory text='Dried Seeds' image={SeedImage} />
-        <FeaturedCategory text='Fruits' image={FruitImage} />
-        <FeaturedCategory text='Nuts' image={NutsImage} />
-        <FeaturedCategory text='Grains' image={GrainsImage} />
+        <div
+          onClick={() => {
+            navigate("/products");
+            filterDispatch({
+              type: "SET_CATEGORY_FROM_HOME",
+              payload: "Super Seeds",
+            });
+          }}
```

Comment:
suggestion: make a separate handler function and perform the operations there

Reply:
Thanks for the suggestion rohit done the changes